src/decision/location_selection.jl

```diff
@@ -28,7 +28,7 @@ function _location_selection(domain::Domain, mcda_vars::DMCDA_vars, guided::Int6
     # Determine connectivity strength
     # Account for cases where no coral cover
     in_conn, out_conn, strong_pred = connectivity_strength(
-        domain.TP_data .* site_k_area(domain), collect(mcda_vars.sum_cover)
+        domain.TP_data .* site_k_area(domain), collect(mcda_vars.prop_cover)
```

Comment:
The collect is because the cover is stored in the domain as a NamedDimsArray. For now I've moved the collect outside into rank_locations(), but open to other ways of doing this